doc/cephfs/standby.rst

```diff
@@ -3,53 +3,50 @@
 Terminology
 -----------
 
-A Ceph cluster may have zero or more CephFS *file systems*.  CephFS
-file systems have a human readable name (set in ``fs new``)
-and an integer ID.  The ID is called the file system cluster ID,
-or *FSCID*.
+A Ceph cluster may have zero or more CephFS *file systems*.  Each CephFS has
+a human readable name (set at creatiopn time with ``fs new``) and an integer
+ID.  The ID is called the file system cluster ID, or *FSCID*.
 
-Each CephFS file system has a number of *ranks*, one by default,
-which start at zero.  A rank may be thought of as a metadata shard.
-Controlling the number of ranks in a file system is described
-in :doc:`/cephfs/multimds`
+Each CephFS file system has a number of *ranks*, numbered beginning with zero.
+By default here is one rank per filesystem.  A rank may be thought of as a
+metadata shard.  Management of ranks is described in :doc:`/cephfs/multimds` .
 
-Each CephFS ceph-mds process (a *daemon*) initially starts up
-without a rank.  It may be assigned one by the monitor cluster.
-A daemon may only hold one rank at a time.  Daemons only give up
-a rank when the ceph-mds process stops.
+Each CephFS ``ceph-mds`` daemon starts without a rank.  It may be assigned one
+by the cluster's monitors. A daemon may only hold one rank at a time, and only
+give up a rank when the ``ceph-mds`` process stops.
 
-If a rank is not associated with a daemon, the rank is
-considered *failed*.  Once a rank is assigned to a daemon,
-the rank is considered *up*.
+If a rank is not associated with any daemon, that rank is considered ``failed``.
+Once a rank is assigned to a daemon, the rank is considered ``up``.
 
-A daemon has a *name* that is set statically by the administrator
-when the daemon is first configured.  Typical configurations
-use the hostname where the daemon runs as the daemon name.
+Each ``ceph-mds`` daemon has a ``name`` that is assigned statically by the
```

Comment:
```suggestion
Each ``ceph-mds`` daemon has a *name* that is assigned statically by the
```

is the correct formatting.

Reply:
We would benefit from some consistent conventions around formatting.  I'd used ``name`` since that's the actual variable, but in the absence of an overarching convention I'll change this occurrence.  This is a bit of a moving target since practice is different in various parts of the doc tree.